Generator/Strategy.ts

```diff
@@ -13,13 +13,19 @@ export enum StrategyEnum {
  * Constructed using 2d array of cells
  * Returns:
  * Whether or object constitutes specific strategies
+ * Cause (cells that "cause" the strategy to be applicable)
  * What candidates can be placed as result of strategy
  * What candidates can be removed from cells notes as result of strategy
+ * What strategy type this is (correlates to StrategyEnum)
  */
 export class Strategy{
     private cells: Cell[][];
```

Comment:
Cells doesn't have a comment and it threw me off 
Maybe "// Contains cells that "cause" strategy to be applicable"